<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dinamik Raporlama Aracı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <item>
     <widget class="QLabel" name="veritabaniLabel">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Veritabanı ve Tablo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Dates">
      <item>
       <widget class="QLabel" name="labelBaslangic">
        <property name="text">
         <string>Başlangıç Tarihi:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date_Baslangic">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBitis">
        <property name="text">
         <string>Bitiş Tarihi:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date_Bitis">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>1</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Dates">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
      <item>
       <widget class="QPushButton" name="btn_Sorgula">
        <property name="text">
         <string>Rapor Al</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Excel">
        <property name="text">
         <string>Excel'e Aktar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_PDF">
        <property name="text">
         <string>PDF'e Aktar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelTaslak">
        <property name="text">
         <string>Rapor Taslağı:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="templateSecCBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Kullanılacak rapor taslağını seçin (Ayarlar menüsünden oluşturulur)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ApplyTemplate">
        <property name="text">
         <string>Taslağı Göster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tarihSecCBox">
        <property name="toolTip">
         <string>Kaydedilmiş Excel raporlarına göz atın</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="geriTarihButton">
        <property name="toolTip">
         <string>Seçili tarihteki önceki Excel raporunu göster</string>
        </property>
        <property name="text">
         <string>&lt; Önceki</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ileriTarihButton">
        <property name="toolTip">
         <string>Seçili tarihteki sonraki Excel raporunu göster</string>
        </property>
        <property name="text">
         <string>Sonraki &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tbl_Veri">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAna_Sayfa">
    <property name="title">
     <string>Ana Sayfa</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyarlar">
    <property name="title">
     <string>Ayarlar</string>
    </property>
    <addaction name="actionTaslak_Duzenle"/>
   </widget>
   <widget class="QMenu" name="menuVeritaban">
    <property name="title">
     <string>Veritabanı</string>
    </property>
    <widget class="QMenu" name="menuVeritaban_T_r_Se_iniz">
     <property name="title">
      <string>Veritabanı Sistemleri</string>
     </property>
     <addaction name="actionMicrosoft_SQL"/>
     <addaction name="actionPostgreSQL"/>
     <addaction name="actionOracle_Database"/>
     <addaction name="actionAccess_Database"/>
     <addaction name="actionMySQL"/>
    </widget>
    <addaction name="menuVeritaban_T_r_Se_iniz"/>
    <addaction name="actionVeritaban_n_Se"/>
   </widget>
   <widget class="QMenu" name="menuProfil">
    <property name="title">
     <string>Profil</string>
    </property>
    <addaction name="actionAdmin_Giri_i"/>
   </widget>
   <addaction name="menuAna_Sayfa"/>
   <addaction name="menuAyarlar"/>
   <addaction name="menuVeritaban"/>
   <addaction name="menuProfil"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVeritaban_n_Se">
   <property name="text">
    <string>Veritabanını Seç / Değiştir...</string>
   </property>
  </action>
  <action name="actionSQL_Server">
   <property name="text">
    <string>SQL Server</string>
   </property>
  </action>
  <action name="actionMicrosoft_SQL">
   <property name="text">
    <string>Microsoft SQL</string>
   </property>
  </action>
  <action name="actionPostgreSQL">
   <property name="text">
    <string>PostgreSQL</string>
   </property>
  </action>
  <action name="actionOracle_Database">
   <property name="text">
    <string>Oracle Database</string>
   </property>
  </action>
  <action name="actionAccess_Database">
   <property name="text">
    <string>Access Database </string>
   </property>
  </action>
  <action name="actionMySQL">
   <property name="text">
    <string>MySQL</string>
   </property>
  </action>
  <action name="actionAdmin_Giri_i">
   <property name="text">
    <string>Admin Girişi</string>
   </property>
  </action>
  <action name="actionTaslak_Duzenle">
   <property name="text">
    <string>Rapor Taslaklarını Yönet...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
